### PR TITLE
docs: fix typos

### PR DESCRIPTION
**Pull request by pgrimaud**
Hey :wave:,

This PR will fix :

1 typo in `src/blog/crytic-continuous-assurance-for-smart-contracts/index.md`
1 typo in `src/blog/the-infura-sdk-nft-truffle-box-all-you-need-to-know/index.md`
1 typo in `src/docs/truffle/reference/contract-abstractions.md`
1 typo in `src/guides/lens-protocol/index.md`
1 typo in `src/guides/rentable-nft/index.md`
1 typo in `src/guides/truffle-and-metamask/index.md`

Best!

### CODE REVIEW

File: src/blog/crytic-continuous-assurance-for-smart-contracts/index.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ Finding an issue is only the first part. Crytic will provide you with detailed i
   <figcaption class="text-center font-italic">Figure 2: Report</figcaption>
 </figure>
 
-A careful reader will notice the vulnerability here: `function constuctor` creates a public function (with a typo!) that is callable by anyone instead of being run only at initialization. Crytic will detect these types of critical mistakes instantaneously.
+A careful reader will notice the vulnerability here: `function constructor` creates a public function (with a typo!) that is callable by anyone instead of being run only at initialization. Crytic will detect these types of critical mistakes instantaneously.
```
Comment (haltman-at) [inline src/blog/crytic-continuous-assurance-for-smart-contracts/index.md:48]: In this case, the typo is deliberate.

**Review by haltman-at — CHANGES_REQUESTED** [review]
Thanks for this!  But the first of these typos is deliberate and should be reverted back to the mistaken way.  Other than that looks good!

**Review by cliffoo — APPROVED** [review]
Thanks! (See Harry's review)

File: src/blog/crytic-continuous-assurance-for-smart-contracts/index.md (hunk 1 above)
Comment (pgrimaud) [reply src/blog/crytic-continuous-assurance-for-smart-contracts/index.md]: Oops, my bad. I just fixed it. 🥴

**Review by haltman-at — APPROVED** [review]

**Comment by haltman-at** [comment]
Thanks again!

**Pull request merged**